lib/logstash/outputs/elasticsearch/common_configs.rb

```diff
@@ -145,8 +135,11 @@ def self.included(mod)
       # if enabled, script is in charge of creating non-existent document (scripted update)
       mod.config :scripted_upsert, :validate => :boolean, :default => false
 
+      # Set initial interval between bulk retries. Doubled on each retry up to `retry_max_interval`
```

Comment:
the unit of time should be described here.
Also as an initial retry interval, I'd suggest something much lower like 0.01 or 0.1 seconds